reference/common_errors.rst

```diff
@@ -171,6 +171,23 @@ Configure the operating system to re-use TCP sockets. Execute this on the shell
 
 This allows reusing sockets in TIME_WAIT state for new connections when it is safe from a protocol viewpoint. Default value is 0 (disabled).
 
+Could not connect to any active placements
+------------------------------------------
+
+When all available worker connection slots are in use, further connections will fail. This happens most often when copying data into Citus in parallel:
```

Comment:
Could we link to http://docs.citusdata.com/en/v7.5/performance/performance_tuning.html#bulk-copy-250k-2m-s? It would also be good to add some color on why this will happen. i.e., COPY always opens up a connection per shard. So if you run 10 concurrent copies, that will result in X number of connections. So recommendation would be to reduce the number of parallel copies.